lib/jekyll-feed/feed.xml

```diff
@@ -39,7 +39,12 @@
     </author>
   {% endif %}
 
-  {% assign posts = site[page.collection] | where_exp: "post", "post.draft != true" | sort: "date" | reverse %}
+  {% if page.tag %}
```

Comment:
What about `tags`?

Reply:
You're right, this is inconsistent. I use "tags" just about everywhere else. Let add a commit to make this more consistent.